components/api_version/src/api_v2.rs

```diff
@@ -16,6 +16,8 @@ pub const RAW_KEY_PREFIX_END: u8 = RAW_KEY_PREFIX + 1;
 pub const TXN_KEY_PREFIX: u8 = b'x';
 pub const TIDB_META_KEY_PREFIX: u8 = b'm';
 pub const TIDB_TABLE_KEY_PREFIX: u8 = b't';
+pub const DEFAULT_KEY_SPACE_ID: [u8; 3] = [0, 0, 0]; // reserve 3 bytes for key space id.
```

Comment:
Why? I thought it should be just one byte?

Reply:
We plan to use 3 bytes fixed length to encode key space id, support more than 16 million key space. And we does not use var int encoding incase there are some issues we may miss.